apis_core/apis_entities/filtersets.py

```diff
@@ -53,6 +56,12 @@ def related_entity(queryset, name, value):
     return queryset.filter(pk__in=t)
 
 
+def changed_since(queryset, name, value):
+    history = get_history_manager_for_model(queryset.model)
+    ids = history.filter(history_date__gt=value).values_list("pk", flat=True)
+    return queryset.filter(pk__in=ids)
```

Comment:
This needs to be changed to:

```suggestion
    ids = history.filter(history_date__gt=value).values_list("id", flat=True)
```

`pk` is the primary key of the version, `id` is the primary key of the db instance

Reply:
sorry my IDE again changed the lines.
line 61 needs to be changed to:

```python
ids = history.filter(history_date__gt=value).values_list("id", flat=True)
```

Reply:
fixed